tensorflow/compiler/aot/codegen.cc

```diff
@@ -3,7 +3,6 @@
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
```

Comment:
You should still keep the empty line. Otherwise license becomes invalid.

Reply:
line added.